models/turbine_models/custom_models/resnet_18.py

```diff
@@ -8,7 +8,7 @@
 from iree.compiler.ir import Context
```

Comment:
Let keep the separate model updates in separate patches. Makes it easier to track and revert patches if ever needed